packages/utils/src/is.ts

```diff
@@ -78,6 +78,22 @@ export function isString(wat: unknown): wat is string {
   return isBuiltin(wat, 'String');
 }
 
+/**
+ * Checks whether given string is parameterized
+ * {@link isParameterizedString}.
+ *
+ * @param wat A value to be checked.
+ * @returns A boolean representing the result.
+ */
+export function isParameterizedString(wat: unknown): wat is ParameterizedString {
```

Comment:
l: Unless I'm missing something, I believe we should also adjust `isString` above. Given that a parameterized string is still a string, no? 

Also, can we add tests for the parameterized string in the adjusted `is` functions? Just wanna make sure we cover these branches.

Reply:
Oh wait, nevermind! it already should return true anyway if we pass a ParameterizedString.